lisa/wlgen/workload.py

```diff
@@ -111,6 +111,37 @@ def wipe_run_dir(self):
         logger.info("Wiping run_dir [%s]", self.run_dir)
         self.target.execute("rm -rf {}".format(quote(self.run_dir)))
 
+    def wrap_command(self, cpus=None, cgroup=None):
+        """
+        Wrap the workload's command with taskset and cgroup constraints
+
+        :param cpus: CPUs on which to restrict the workload execution (taskset)
+        :type cpus: list(int)
+
+        :param cgroup: cgroup in which to run the workload
+        :type cgroup: str
+
+        :returns: The command augmented with relevant taskset/cgroup invocations
+        """
+        command = self.command
+
+        if not command:
+            raise RuntimeError("Workload does not specify any command to execute")
+
+        if cpus:
+            taskset_bin = self.target.which('taskset')
+            if not taskset_bin:
+                raise RuntimeError("Could not find 'taskset' executable on the target")
```

Comment:
since this method is the one deciding which binaries are going to get used, it would make sense to `self.target.install()` rather than just complaining to the caller that the binary is not there.

Reply:
Hmm actually it's already installed at `__init__()` time since `taskset` is in the list of `required_tools` for the base `Workload` class. We've always had this safety check, I suppose I could remove it?